documentation/rfc/ptref_grammar.md

```diff
@@ -0,0 +1,225 @@
+# Public Transit Referential grammar
+
+It's possible to do advanced query to the public transit referential. The query is in the form `/v1/coverage/id/collections?filter=query`. This is the documentation of the `query` syntax. For simplicity, the queries in this documentation will not be urlencoded, but it must be in practice.
```

Comment:
Minor:
"This is the documentation of the `'collections' and `'query' syntax. `'query' must be urlencoded in practice, but for simplicity it is not the case in this documentation.`"